doc/ug/getting_started_hw_design.md

```diff
@@ -90,3 +90,47 @@ To do so, follow the steps below.
 * Open a Pull Request with the necessary code changes.
 
 If it is not clear on how to proceed, feel free to file an issue requesting assistance.
+
+## Change of Interrupts
+
+Changing the interrupts of an IP block needs some extra work in addition to the RTL change.
+1. If a DV environment exists, its testbench (`tb.sv`) will need the ports connected.
+1. If the IP block has been instantiated in a top-level, that top-level will need re-generating to include the new ports.
+1. The auto-generated DIFs will need re-generating.
+1. The PLIC test should also be updated.
+
+Item 1 is reasonably self-explanatory.
+The sections below show how to do items 2-4.
+
+### Update TOP
+
+```console
+$ make -C hw top
+```
+
+After you revise the IP `.hjson`, IP top module, IP register interface, and DV testbench `tb.sv`, you can re-generate top-levels with above command.
```

Comment:
Grammar nit: I think you want either "with the above command" or (better) "with the command above" (missing "the")
```suggestion
After you revise the IP `.hjson`, IP top module, IP register interface, and DV testbench `tb.sv`, you can re-generate top-levels with the command above.
```